<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataInfoAreaComponent</class>
 <widget class="QMainWindow" name="CDataInfoAreaComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="qw_centralEmptyWidgetUsedAsPadding">
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>3</height>
    </size>
   </property>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_AircraftModels">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Aircraft models</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_AircraftModelsOuter">
    <layout class="QVBoxLayout" name="vl_AircraftModelsOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_AircraftModelsInner">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="vl_AircraftModelsInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbModelComponent" name="comp_DbModels">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_Liveries">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Liveries</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_LiveriesOuter">
    <layout class="QVBoxLayout" name="vl_LiveriesOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_LiveriesInner">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="vl_LiveriesInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbLiveryComponent" name="comp_DbLiveries">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_Distributors">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Distributors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_DistributorsOuter">
    <layout class="QVBoxLayout" name="vl_DistributorsOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_DistributorsInner">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="vl_DisitributorsInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbDistributorComponent" name="comp_DbDistributors">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_AircraftIcao">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Aircraft ICAO</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_AircraftIcaoOuter">
    <layout class="QVBoxLayout" name="vl_AircraftIcaoOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_AircraftIcaoInner">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="vl_AircraftIcaoCodeinner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbAircraftIcaoComponent" name="comp_DbAircraftIcao">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_AirlineIcao">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Airline ICAO</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_AirlineIcaoOuter">
    <layout class="QVBoxLayout" name="vlAirlineIcaoOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_AirlineIcaoInner">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vl_AirlineIcaoInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbAirlineIcaoComponent" name="comp_DbAirlineIcao">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoArea" name="dwp_Countries">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Countries</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_Countries">
    <layout class="QVBoxLayout" name="vlAirlineIcaoOuter_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_Countries">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vl_AirlineIcaoInner_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BlackGui::Components::CDbCountryComponent" name="comp_DbCountries">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CDockWidgetInfoArea</class>
   <extends>QDockWidget</extends>
   <header>blackgui/dockwidgetinfoarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbModelComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbmodelcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbLiveryComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbliverycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbDistributorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbdistributorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAircraftIcaoComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbaircrafticaocomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineIcaoComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlineicaocomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbCountryComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbcountrycomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
